.changeset/gorgeous-days-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
```

Comment:
@alex-ju based on your survey, should this be a patch or a minor?

Reply:
the survey was somewhat nonconclusive on this (at least not a consensus). minor seems appropriate to me.

Reply:
Yeah, I think @andgen404 will communicate this more widely on slack before we merge so people know its happening.